styles/_sub-nav.scss

```diff
@@ -19,4 +19,7 @@
 
     &::after { font-size: $font-size-base; }
   }
+  @media print {
```

Comment:
missing a space before this?

Reply:
No. it doesn't seem to want space before media queries.